src/modules/init/index.ts

```diff
@@ -15,10 +15,7 @@ const currentFolderName = process.cwd().replace(/.*\//, '')
 const templates = {
   'react getting-started': 'render-getting-started',
   'graphql getting-started': 'product-review-graphql-example',
-  'react+graphql': 'catalogue',
-  'hello graphql': 'hello-graphql',
-  'hello react': 'hello-react',
-  'dreamstore getting-started': 'dreamstore-getting-started',
+  'react+graphql': 'render-guide',
```

Comment:
VTEX is very against merge commits, so you should probably rebase from master instead of adding a merge commit.

Reply:
I accidentally made the change in the browser. I'll rebase instead